app/workers/builds.php

```diff
@@ -183,7 +183,6 @@ protected function buildDeployment(Document $project, Document $function, Docume
 
             /* Also update the deployment buildTime */
             $deployment->setAttribute('buildTime', $response['duration']);
-            $deployment = $dbForProject->updateDocument('deployments', $deployment->getId(), $deployment);
```

Comment:
The two lines above should be removed too, but I'm not sure where the commit for that went 🧐

Reply:
Not sure... synced multiple times with those branches but it doesnt seem to fix it